tst/EnergyPlus/unit/BoilerHotWater.unit.cc

```diff
@@ -71,6 +71,7 @@ using namespace EnergyPlus::Psychrometrics;
 
 TEST_F(EnergyPlusFixture, Boiler_HotWaterSizingTest)
 {
+    state->init_state(*state);
```

Comment:
Do it like this? Or selectively call `init_state` on the subset of modules you are using in the test?

Reply:
That may be the better approach for unit testing. However, there would be several init_state calls to set everything up.